tests/www/companies_views/__snapshots__/test_views.ambr

```diff
@@ -287,6 +287,31 @@
               
       <section class="s-section">
           <div class="s-section__container container">
+              <div class="row">
+                  <div class="col-12">
+                      <div class="c-box c-box--action mb-4 hidden-if-empty">
+                          <h2 class="visually-hidden">Actions rapides</h2>
+                          <div class="form-row align-items-center gx-3">
+                              
+                                  <div class="form-group col col-lg-auto">
+                                      <a aria-label="Postuler auprès de l'employeur solidaire Acme inc." class="btn btn-lg btn-white btn-block btn-ico" data-matomo-action="clic" data-matomo-category="candidature" data-matomo-event="true" data-matomo-option="start_application" href="/apply/85065/start">
```

Comment:
`/apply/85065/start` contient la PK de la SIAE, qui changera à chaque lancement des tests.
Il faudrait donc utiliser l'argument `replace_in_href=` de `assertSoupEqual` dans le test concerné.

Reply:
Bon là je suis paumé. Ça a peut etre a voir avec ce test `tests/www/companies_views/test_views.py::CardViewTest::test_card_no_active_members` qui plante sur un snapshot alors que le `pytest --snapshot-update` ne dit que tout est ok

Reply:
C'est le bon test et dans le `parse_response_to_soup` (https://github.com/gip-inclusion/les-emplois/blob/master/tests/www/companies_views/test_views.py#L58) tu peux rajouter un:
```
replace_in_href=[
                (f"/apply/{company.pk}/start", "/apply/[PK of Company]/start"),
            ],
```
(regarde les autres utilisations du `replace_in_href` dans les test autour)

Reply:
C'est fait mais ça plante encore...

Reply:
Oui, ensuite il faut regénérer le snapshot pour qu'il contienne `/apply/[PK of Company]/start` et non plus `/apply/87768/start`.